docs/building-spark.md

```diff
@@ -214,7 +214,7 @@ For help in setting up IntelliJ IDEA or Eclipse for Spark development, and troub
 
 # Running Tests
 
-Tests are run by default via the [ScalaTest Maven plugin](http://www.scalatest.org/user_guide/using_the_scalatest_maven_plugin).
+Tests are run by default via the [ScalaTest Maven plugin](http://www.scalatest.org/user_guide/using_the_scalatest_maven_plugin). All tests should not be run by `root` or any admin user.
```

Comment:
I'm OK with it though might suggest slightly different wording, like "note that tests should not be run as `root` or an admin user."